Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/SelectionComponent.cpp

```diff
@@ -11,14 +11,14 @@
 #include <AzCore/Memory/Memory.h>
 #include <AzCore/Serialization/Json/BaseJsonSerializer.h>
 #include <AzCore/Serialization/Json/RegistrationContext.h>
+#include <AzCore/Component/EntitySerializer.h>
 
 namespace AzToolsFramework
 {
     namespace Components
     {
-        static constexpr const char* const SelectionComponentRemovalNotice =
-            "[INFORMATION] The editor SelectionComponent is being removed from .prefab data. Please edit and save the level to persist "
-            "the change.";
+        static constexpr const char* const SelectionComponentLoadMessage =
+            "The editor SelectionComponent is being skipped during loading because it has been deprecated.";
```

Comment:
suggestion: It might also be worth adding 'and will be removed from the .prefab file the next time it is saved'... or something like that to indicate it's actually being physically removed not just indefinitely skipped?

Reply:
Updated. Note that this message will not currently be displayed to the user because the skip operation does not translate to a warning or error, and currently the default Json reporter only outputs warnings/errors.

Reply:
Okay understood thanks!